### ISSUE TITLE
Current status and API documentation

### ISSUE THREAD

**Issue by ClaesNilsson** [vicktor/FreeStyleLibre-NFC-Reader#9]
Found this repo. Looks indeed interesting!

I have two questions:

1. Which is the current status of this project? Is it up to date with the current sensors from Abbott?

2. Where can I find the API documentation from Abbott?

Thanks

**Comment by vicktor** [comment]
its a proof of concept, never use it alone

there is no api from Abbott, no help from Abbott, no relation with Abbott

--
Victor

http://www.linkedin.com/in/victorbautista
http://twitter.com/0x564242

**Comment by ClaesNilsson** [comment]
Ok, thanks Vicktor. Sad to hear that you didn't get any help from Abbott. I assume that they make their business on the selling the sensors, not the readers. In this world of open source the ability for 3rd parties to create applications reading the Libre sensors would boost their business.

**Comment by Geoneer** [comment]
indeed.... I want to reactivate the sensor after 14 days. Is that possible?
